packages/orchestrator-ui-components/src/components/WfoTable/WfoTableSettingsModal/WfoTableSettingsModal.tsx

```diff
@@ -71,15 +77,14 @@ export const TableSettingsModal = <T,>({
                 })
             }
         >
+            {/* Todo not sure how to target the text in the form - it needs a better color */}
```

Comment:
Any progress on this? Should it be a ticket?

Reply:
whoops accidental commit, i found it haha